packages/modular-scripts/src/cli.ts

```diff
@@ -180,16 +183,22 @@ program
 interface InitOptions {
   y: boolean;
   preferOffline: string;
+  verbose: boolean;
 }
 
 program
   .command('init')
   .description('Initialize a new modular root in the current folder')
   .option('-y', 'equivalent to the -y flag in NPM')
   .option('--prefer-offline [value]', 'delegate to offline cache first', true)
+  .option('--verbose', 'Run yarn commands with --verbose set')
```

Comment:
We should set `process.env.MODULAR_LOGGER_DEBUG` to true if they want verbose logs.